config/checkstyle-non-main-files-suppressions.xml

```diff
@@ -203,11 +203,7 @@
 
   <!-- no properties in module by design -->
   <suppress id="propertiesMacroMustExist"
```

Comment:
There is too much going on in this  file. Please split changes between #13693 and others to some minor or such commit.

Reply:
@rnveach , please rephrase I do not understand.
I am going by two directions from top and from bottom. There is no other activity.

Reply:
I keep 2 block of permanent suppressions due to no properties on top and bottom to ease diff, at the end they will naturally combined

Reply:
It is hard to read this file with the changes going on. Some look related to https://github.com/checkstyle/checkstyle/issues/13693 , but then there are some surrounding 13839 , 13859 , 13860 as they specifically mention those issues. I take those other issues ones not related to 13693 .

Any changes pertaining to 13693 , should go in 1 commit, everything else should go into another for this one file.

Reply:
@rnveach , done.